docs/_docs/tools/control-script.adoc

```diff
@@ -646,4 +646,136 @@ control.bat --change-tag _<new-tag>_
 ----
 --
 
+== Metric command
 
+Metrics command prints value(s) of the metric or metric registry from the parameter.
+
+If you need to get metric from the specific node, please, set the `--node-id` parameter.
+Without `--node-id` random node will be used.
+
+[tabs]
+--
+tab:Linux[]
+[source,shell,subs="verbatim,quotes"]
+----
+control.sh --metric sys
+----
+tab:Windows[]
+[source,shell,subs="verbatim,quotes"]
+----
+control.bat --metric sys
+----
+--
+
+Example of the metric output:
+[source, text]
+$ ./bin/control.sh --metric sysCurrentThreadCpuTime
```

Comment:
Remove the command row from the output here and similar places below. Let's follow the same format you (we) have already used for other control script commands documented earlier.